spec/tic_tac_toe/board_spec.clj

```diff
@@ -4,4 +4,15 @@
 
 (describe "a board"
   (it "returns an empty board"
-    (should= '(- - - - - - - -) (empty-board) )))
+    (should= ["-" "-" "-" "-" "-" "-" "-" "-" "-"]
+             (empty-board) ))
+  (it "makes a move in the center"
+    (should= ["-" "-" "-" "-" "X" "-" "-" "-" "-"]
+             (make-move ["-" "-" "-" "-" "-" "-" "-" "-" "-" ] 4 "X") ))
+  (it "makes a move in the top right"
+    (should= ["-" "-" "-" "-" "X" "-" "-" "-" "O"]
+             (make-move ["-" "-" "-" "-" "X" "-" "-" "-" "-" ] 8 "O") ))
+  (it "does not make a move for out for invalid move"
```

Comment:
The test title doesn't make sense here - `doesn't make a move when outside the board` or `doesn't make an invalid move` would be clearer.